data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,103 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	(Try and find another way in.)`
+					goto attwr
+				`	(Help the others bust the gate open.)`
+			`	You start pushing with the others, the Saryds joining you below, and the Kimek and Arachi crawling up to push the top of the gate. They take turns with others when they get tired so that someone is always pushing, even though their attempts aren't the most effective. After a while, you hear a cracking noise, and the gate begins budging slightly, as if something barricading it had broken off. As the latecomers of the main force come up, bringing with them the battering ram, everyone makes way for them to begin hammering it on the gate. It takes some good damage with each time, gradually budging more. The battering ram cannot hit it enough times, however, as the whistle from the start of the event sounds again. It evidently means that time has run out and that the defenders have won.`
+				goto lost
+			label attwr
+			`	You and a part of the group stick close to the walls and follow along them to the other side, careful not to scratch yourself on the thorns along the fort. You soon find another, smaller gate, and begin pushing. Unlike the main gate, you can feel it budging, and once they realize there aren't many people guarding this gate, the Saryds in the group ask others to move out of the way. You are a bit startled as they suddenly charge at it, almost forcing it open. More defenders come to hold it, though, and the gate becomes increasingly difficult to budge. One Arach heads back to the main group to call for more help, and when they arrive, so do more defenders. The struggle continues for many minutes, with the attackers unable to make a breakthrough. The whistle sounds again, marking the end of the battle, announcing that the defenders have won.`
+				goto lost
+			label attr
+			`	You follow a group to the edges of this set, losing some time before beginning the run up the hill. Once you finally start making your way up, you turn to see the main group tripping and falling back. It seems the defenders are pouring some slippery liquid down from the battlements above the gate, slowing down those that are coming from that path. You reach the top while the main force still struggles, but your group still makes its way to the gate. On your way, you see what the Kimek meant when she spoke of "thorns," with closely-spaced tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls. You reach the gate, but as you push and push, it refuses to budge even the slightest bit, having clearly been barricaded from the inside. The main group eventually catches up to you and a lot more people begin pushing now, though it still resists firmly.`
```

Comment:
```suggestion
			`	You follow a group to the edges of this set, losing some time before beginning your run up the hill. Once you finally start making your way up, you turn to see the main group repeatedly tripping and falling back. It seems the defenders, who are wearing green identifiers instead of blue, are pouring some slippery liquid down from the battlements above the gate, slowing those that are coming along a direct path. You reach the top while the main force still struggles, and your group makes its way to the gate. On your way, you see what the Kimek meant when she spoke of "thorns," with closely-spaced tufts of vicious, reddish barbs nearly as long as your arm covering the logs that make up the walls. You reach the gate, but as you push and push, it refuses to budge even the slightest bit, clearly having been barricaded from the inside. The main group eventually catches up to you, but even with many more people pushing now, the gate still holds firm and unmoving.`
```